packages/scoutgame/src/points/calculateBuilderLevel.ts

```diff
@@ -24,12 +24,23 @@ export const decileTable = [
   { cutoff: 0, level: 1 }
 ];
 
-export async function calculateBuilderLevels({
-  season = getCurrentSeasonStart()
-}: {
-  season?: ISOWeek;
-}): Promise<BuilderAggregateScore[]> {
-  const allSeasonWeeks = getAllISOWeeksFromSeasonStart({ season });
+export async function calculateBuilderLevels(
+  {
+    season = getCurrentSeasonStart()
+  }: {
+    season?: ISOWeek;
+  } = {
+    season: getCurrentSeasonStart()
```

Comment:
i dont think you need to set the default here, you can just do `= {}`

Reply:
Nice, thanks